raiden/transfer/state.py

```diff
@@ -351,7 +351,7 @@ class PaymentNetworkState(State):
     __slots__ = (
         'address',
         'tokenidentifiers_to_tokennetworks',
-        'tokenaddresses_to_tokennetworks',
+        'tokenaddresses_to_tokenidentifiers',
```

Comment:
from a quick grep I only find 9 used of this. I think it would make sense to get rid of this indirection all together.
```
± % rg tokenaddresses_to_tokenidentifiers                                                       14:52:47
raiden/transfer/views.py
193:            for token_address in payment_network.tokenaddresses_to_tokenidentifiers.keys()
242:        token_network_id = payment_network.tokenaddresses_to_tokenidentifiers.get(token_address)

raiden/transfer/node.py
78:        token_network_id = payment_network_state.tokenaddresses_to_tokenidentifiers.get(
394:        addrs_to_ids = payment_network_state.tokenaddresses_to_tokenidentifiers
507:            del payment_network_state.tokenaddresses_to_tokenidentifiers[

raiden/transfer/state.py
354:        'tokenaddresses_to_tokenidentifiers',
370:        self.tokenaddresses_to_tokenidentifiers = {
382:            self.tokenaddresses_to_tokenidentifiers == other.tokenaddresses_to_tokenidentifiers and

raiden/tests/unit/fixtures.py
61:    payment_network_state.tokenaddresses_to_tokenidentifiers[token_id] = token_network_id
```

Reply:
sure, but I would prefer to do that in another PR, the reason for the current style is mostly a mismatch from the REST API / internals / Smart Contracts

Reply:
Yeah sure, I thought that was the only indirection, but the ones below require more work.